src/components/icon-card-link/index.tsx

```diff
@@ -0,0 +1,28 @@
+import { CSSProperties } from 'react'
+import CardLink from 'components/card-link'
+import { IconCardLinkProps } from './types'
+import s from './icon-card-link.module.css'
+
+const IconCardLink = ({ icon, productSlug, text, url }: IconCardLinkProps) => {
+  let colorToken: string
+  if (productSlug === 'sentinel' || productSlug === 'hcp') {
+    colorToken = '--token-color-hashicorp-brand'
```

Comment:
🎨 Optional 🎨  We could set a classname here instead of handling the styling via JSX? To maintain css separation where possible.

Reply:
I think the reason we didn't is cause of the more dynamic `--token-color-${productSlug}-brand`. I believe we'd need _many_ class names!

Reply:
I also think'd we need to add many classnames to accomplish this, and if we decide to go that route I'd prefer to see them as `g-` helpers that are usable throughout the app. 😊

I don't have a preference over setting variables or not in the JSX, we do already have a few places we're doing it now. Sounds very worthy of a team discussion to get some more opinions and context!